### ISSUE TITLE
[BUG] Can only convert one Fissure Stone to Valor at a time

### ISSUE THREAD

**Issue by ChristianBlom84** [ClassicWoWCommunity/cata-classic-bugs#1559]
#### 1.1. Description
If you try to convert more than 1 Fissure Stone to Valor Points at a time you get the error message: "You can't buy the specified quantity of that item."
  #### 1.2. How to Reproduce
Try to buy any quantity of VP for Fissure Stone higher than 10.
#### 2.1. Description
You should be able to convert all your Fissure Stones to Valor at the same time.

**Comment by Beakachu** [comment]
Non-issue there are scripts you can run to exchange multiple stones.

2 I found from Google. Haven't tested these so run at your own risk.

/run for i=1,GetMerchantNumItems() do local name=GetMerchantItemInfo(i) if name=="Valor Points" then BuyMerchantItem(i, 1) break end end

/script BuyMerchantItem(94)

**Comment by ChristianBlom84** [comment]
So the reason to not fix a bug is that you can Google a work-around? They've fixed bugs like these before. That was in Wrath though, when they still cared about Classic, so maybe not.
